### PR TITLE
Do cfitsio410 migration and other touchups

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #69.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,15 +52,15 @@ requirements:
     - libblas
     - libcblas
     - liblapack
-    - boost {{ boost }}
-    - cfitsio {{ cfitsio }}
-    - fftw {{ fftw }}
-    - gsl {{ gsl }}
-    - hdf5 {{ hdf5 }}
-    - ncurses {{ ncurses }}
```
Comment (hmaarrfk) [inline recipe/meta.yaml:60]: Any reason for this strange syntax before? is it legacy?

**Review by hmaarrfk — COMMENTED** [review]
not sure why the harsh pinnings were in place but this should help the migrations moving forward

File: recipe/meta.yaml (hunk 1 above)
Comment (pkgw) [reply recipe/meta.yaml:60]: Yes, legacy.

**Review by pkgw — APPROVED** [review]
Looks good, thanks! Were there any other changes that you wanted to include here? If not I can go ahead and merge.

**Comment by hmaarrfk** [comment]
No this is it.

**Pull request merged**